generator/codemods/router/__testfixtures__/create-router.input.js

```diff
@@ -19,4 +19,13 @@ const router = new VueRouter({
   routes
 });
 
+new VueRouter({
+  routes
+});
+
+new VueRouter({
```

Comment:
I don't know if this test makes sense because I've never used jscodeshift before so I don't know if different node types are worth testing

Reply:
Not necessary, as we've taken the whole `property.value` regardless of its type.

Reply:
Ok! I removed them!